### ISSUE TITLE
next build crashes if rewrites object contains just the `fallback` key

### ISSUE THREAD

**Issue by klapec** [axiomhq/next-axiom#21]
I'm having a trouble integrating axiom within my existing nextjs (@12.2.2) app. Looks like `withAxiom` throws when `next.config.js` contains a following rewrites configuration:
```javascript
async rewrites() {
    return {
      fallback: [
        {
          source: '/:region/:name([\\w-]{1,})-:id(\\d{1,})',
          destination: '/activity-card/:name/:id',
        },
      ],
    };
  },
  ```
with the exact error message being
```
xxx:build: > Build error occurred
xxx:build: TypeError: Cannot read properties of undefined (reading 'concat')
xxx:build:     at /Volumes/Dev/repos/yyy/node_modules/next-axiom/dist/withAxiom.js:88:71
xxx:build:     at step (/Volumes/Dev/repos/yyy/node_modules/next-axiom/dist/withAxiom.js:44:23)
xxx:build:     at Object.next (/Volumes/Dev/repos/yyy/node_modules/next-axiom/dist/withAxiom.js:25:53)
xxx:build:     at fulfilled (/Volumes/Dev/repos/yyy/node_modules/next-axiom/dist/withAxiom.js:16:58)
xxx:build: error Command failed with exit code 1.
```

Looking at the `withAxiom` source code https://github.com/axiomhq/next-axiom/blob/f5be9ade99f0de80ce4918908d1cf801e1c68dbc/src/withAxiom.ts#L40 I see that only `afterFiles` field is supported. Why is that so?

**Comment by bahlo** [comment]
Hi @klapec, thanks for the issue, fixed in #22, will be in `v0.11.0` which will be released next week 👍

**Comment by klapec** [comment]
Awesome, thank you!
